src/library/scala/collection/immutable/BitSet.scala

```diff
@@ -60,6 +60,8 @@ abstract class BitSet extends scala.collection.AbstractSet[Int]
       updateWord(idx, word(idx) & ~(1L << elem))
     } else this
   }
+  @transient override lazy val size = super.size
+  @transient override lazy val hashCode = super.hashCode
```

Comment:
If a `lazy val` is marked `@transient`, does that mean we don't need to worry about it at deserialization time? Is the lazy bit cleared, so that the values will be recomputed on demand?

Reply:
in my experience with hadoop, making something a transient lazy meant it would not be serialized, and then on the receiving end it would be recomputed. I don't know if transient on lazy was ever specified, but that is how it behaved up to scala 2.12 afaik.